src/test/loader.ts

```diff
@@ -216,7 +227,7 @@ function toReporters(reporters: 'dot' | 'line' | 'list' | 'junit' | 'json' | 'nu
 }
 
 function errorWithFile(file: string, message: string) {
-  return new Error(`${file}: ${message}`);
+  return new Error(`${path.relative(process.cwd(), file)}: ${message}`);
```

Comment:
I think we should use there the project rootDir which is stored inside the config. At least thats what I did with other related error reporting.

Reply:
Undid this because its controversial.